### ISSUE TITLE
Update Import Scripts (Roster)

### ISSUE THREAD

**Issue by jordanngtw** [PublicDataWorks/complaint-manager#233]
### Business Context:
~blocked by #234~

Need to be able to upload roster of people in custody so that HCSOC can select complainants

**As a** dev
**I want** to upload a roster
**So that** I can use the roster to populate fields

### In Scope
1. Update scripts
2.
3.

### Out of Scope
1. search for person in roster
2. 
3.

### Open Questions
1.
2.
3.

## Acceptance Criteria

1. Scenario: Able to add roster to 
Given a roster from hawaii
When the roster is uploaded
Then the info is in the DB

2. Scenario: no duplicates
Given an update to the hawaii roster
When the update is uploaded and the job is rerun
Then the database does not contain duplicates of people who were in previous rosters

### Checklist (should be reviewed before grooming as the BA or Pair are writing the card—NOT reviewed for the first time at grooming)
0. Discuss the technical implementation of this card.
1. Discuss how this feature would be included in or affect the Audit Log.
2. In respect to security, what could go wrong? Brainstorm threats using STRIDE model, especially if this card introduces a new technical architecture or design pattern. (TIMEBOX: 7 mins) https://docs.google.com/document/d/18AFl1d3SRieI4d_8R3GyEazqQOycLNDHGN_vOYIHuiE/edit#heading=h.aio9zb5j63wl
3. Ensure any open questions have been resolved with stakeholders.
4. Determine if documentation should be referenced or created.
5. Ensure to include all necessary mockups to this card.
6. Discuss what success/failure user feedback messages would be present, if any occur (front end/back end).
7. Discuss why this feature would not be implemented behind a toggle / what would happen if this card was partially done while a release branch was being cut.
8. If this card is part of a sequence of cards, discuss if this card should be blocked until the testing of previous  cards in the sequence have been finished.
9. Have we added edge cases as ACs that we're considered from a QA perspective
10. Does this card overlap with any entities outlined in [API Housekeeping](https://www.pivotaltracker.com/story/show/173963545)?  

**Notes:**

**Comment by gumidyal** [comment]
[Dummy Hawaii Roster - Sheet1.csv](https://github.com/PublicDataWorks/complaint-manager/files/10361970/Dummy.Hawaii.Roster.-.Sheet1.csv)
